requirements_dev.txt

```diff
@@ -7,6 +7,6 @@ tox==3.14.0
 coverage==4.5.4
 Sphinx==1.8.5
 twine==1.14.0
-webexteamssdk==1.6.1
+webexpythonsdk
```

Comment:
Please pin to latest version

Reply:
Pointed webexpythonsdk to the latest version (2.0.2)